driver/UFS.F90

```diff
@@ -0,0 +1,515 @@
+#define ESMF_ERR_ABORT(rc) if (ESMF_LogFoundError(rc, msg="Aborting NEMS", line=__LINE__, file=__FILE__)) call ESMF_Finalize(endflag=ESMF_END_ABORT)
+!-----------------------------------------------------------------------
+!
+      PROGRAM UFS
```

Comment:
Is there a reason why we keep using  fixed format in our codes.  This is not F77 anymore.

Reply:
Same comment on the other file `driver/EARTH_GRID_COMP.F90` added.

Reply:
It is not F77, this is F90. It's also in free format.